File: app/src/main/java/com/magnitudestudios/GameFace/pojo/EmitMessage.java
package com.magnitudestudios.GameFace.pojo;

public class EmitMessage {
    public String userID;
    public String type;
    public Object data;

    public EmitMessage() {

    }

    public EmitMessage(String user, String type, Object data) {
        this.userID = user;
        this.type = type;
        this.data = data;
    }
}

File: app/src/main/java/com/magnitudestudios/GameFace/pojo/EmitMessage.kt
package com.magnitudestudios.GameFace.pojo

data class EmitMessage(
        @JvmField
        val userID: String,
        @JvmField
        val type: String,
        @JvmField
        val data: Any? = null
) {
    constructor() : this("","", "")
}